From c7a172517deb7ca8920b6c910998cfecd8b1893d Mon Sep 17 00:00:00 2001
From: Mathieu Duponchelle <mathieu.duponchelle@opencreed.com>
Date: Tue, 13 Jun 2017 18:51:32 +0200
Subject: [PATCH] gstgdkpixbufdec: stop pretending to decode gifs.

If you can't decode an animated gif, you can't decode a gif,
so stop squatting GST_RANK_SECONDARY for that format, libav
does a better job.

https://bugzilla.gnome.org/show_bug.cgi?id=784683
---
 ext/gdk_pixbuf/gstgdkpixbufdec.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ext/gdk_pixbuf/gstgdkpixbufdec.c b/ext/gdk_pixbuf/gstgdkpixbufdec.c
index 1598c53be..09211bf17 100644
--- a/ext/gdk_pixbuf/gstgdkpixbufdec.c
+++ b/ext/gdk_pixbuf/gstgdkpixbufdec.c
@@ -38,7 +38,7 @@ static GstStaticPadTemplate gst_gdk_pixbuf_dec_sink_template =
     GST_PAD_ALWAYS,
     GST_STATIC_CAPS ("image/png; "
         /* "image/jpeg; " disabled because we can't handle MJPEG */
-        "image/gif; "
+        /*"image/gif; " disabled because we can't handle animated gifs */
         "image/x-icon; "
         "application/x-navi-animation; "
         "image/x-cmu-raster; "
-- 
2.15.1

